python/lib/cloudutils/test.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+from .utilities import bash
+import logging
+
+cmd = bash("route -n|awk \'/^0.0.0.0/ {print $2,$8}\'")
+if not cmd.isSuccess():
+    logging.debug("Failed to get default route")
+    raise CloudRuntimeException("Failed to get default route")
+
+result = cmd.getStdout().split(" ")
```

Comment:
@davidjumani @shwstppr This looks like a new file, if this was for testing while developing the feature and is unecessary (not tied or run with travis/trillia) - remove it please.

Reply:
removed